.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
- Add EOF character.
- Always remember that code does not ends at last line of file.